Expanding Lobster's Cyrillic

alexeiva opened the pull request:
@ThomasJockin

Hey Thomas,

I am expanding Lobster's Cyrillic. Please don't merge this request yet, until all the work in complete. 

Cheers, 
-a

m4rc1e committed:
deleted legacy files

m4rc1e committed:
Lobster.glyphs: font now passes Font Bakery

m4rc1e committed:
otf: generated otf version

m4rc1e committed:
.DS_Store: rm chached version

m4rc1e committed:
gitignore: added

m4rc1e committed:
.DS_Store: rm chached version

m4rc1e committed:
OFL: Changed first line

m4rc1e committed:
ttf: regenerated ttf

m4rc1e committed:
DS_Store: removed

m4rc1e committed:
sources: Fixed v metrics

m4rc1e committed:
DESCRIPTION.en_us.html: Readded description html file

m4rc1e committed:
Updated font metadata to match old METADATA.pb files

alexeiva committed:
Merge pull request #1 from m4rc1e/master

merging in Marc's updates

alexeiva committed:
CYR Plus:  added descender glyphs

alexeiva committed:
fixed ogonek vs cedilla. Placed anchors on comb marks

alexeiva committed:
working on descenders

alexeiva committed:
Added glyphs Ҥҥѣ, and others

alexeiva committed:
added glyphs ЏѢѪѴҒҔҘҜҠҪҸѫғҕҗҙҝҡҫҹӷ₽. Rebuilt Cyrillic compoennts

m4rc1e committed:
Merge pull request #1 from cyrealtype/master

Expanding Lobster's Cyrillic

alexeiva committed:
adding localised variants

alexeiva committed:
checked and fixed v metrics — win Descent from 250 to 540

alexeiva committed:
updated Cyrillic descenders to script-like, updated ү

alexeiva committed:
updated v metrics

alexeiva committed:
updating kerning classes

alexeiva committed:
new design for uc descnders in ҢԤҶӶ

alexeiva committed:
ran Toshe's kern classes script on Cyrillic, and revised all exisintg CYR kern classes

m4rc1e committed:
Merge pull request #2 from cyrealtype/master

Expanding Lobster's Cyrillic

alexeiva committed:
improving Cyrillic kerning uc. in progress

alexeiva committed:
improvements for uc Cyrillic kerning complete

alexeiva committed:
added U/l Cyrillic kerning

alexeiva committed:
added kernign classes to new Cyr glyphs. Updates features. Added /yi-cy.end

alexeiva committed:
minor fixes. generated fonts.

alexeiva committed:
improving lc-lc CYR kernign. w-i-p

alexeiva committed:
improvements for lc to lc CYR kernign complete

alexeiva committed:
all Marc's QA check's passed. Generated fonts

alexeiva committed:
ran fontbakery checks. updated windesent as suggested to 490

alexeiva committed:
fixed NULL and fstype fontbakery errors thanks to Marc's updated script, and Glyphs cutting edge version

m4rc1e committed:
Merge pull request #3 from cyrealtype/master

kerning improvements

m4rc1e committed:
OFL.txt: updated copyright string

m4rc1e committed:
sources | fonts: copyright string and name table tidied

Commit:
fixed fractions, percent, kerned dnom and numr figures for dynamic fractions

alexeiva committed:
fixed fractions, percent, kerned dnom and numr figures for dynamic fractions

m4rc1e committed:
Merge pull request #4 from alexeiva/master

fixed fractions, percent, kerned dnom and numr figures

alexeiva commented:
@m4rc1e is this ready for merging now?

m4rc1e commented:
@alexeiva I need to do a once over myself. 

I'll send him a pr and close this one.

Cheers,
Marc

alexeiva commented:
@m4rc1e good. thank you

impallari committed:
Merge pull request #5 from m4rc1e/master

v2.000: mastering & Cyrillic kerning updated

alexeiva committed:
Merge branch 'master' of https://github.com/alexeiva/The-Lobster-Font

alexeiva committed:
Merge https://github.com/impallari/The-Lobster-Font

alexeiva committed:
fixed %

impallari committed:
Merge pull request #7 from alexeiva/master

fixes #6 percent

m4rc1e committed:
sources | fonts | OFL: Updated copyright string. RFN moved onto first line

m4rc1e committed:
otf: generated

impallari committed:
Merge pull request #8 from m4rc1e/master

Updated copyright string & generated otf

alexeiva committed:
Merge pull request #1 from m4rc1e/master

sync with Marc

alexeiva committed:
renamed ze_ze to dze_ze to fix https://github.com/google/fonts/issues/525 part 1

alexeiva committed:
added зз ligature to improve https://github.com/google/fonts/issues/525

m4rc1e committed:
Merge pull request #5 from alexeiva/master

Urgent fix for https://github.com/google/fonts/issues/525

m4rc1e committed:
Merge branch 'master' of https://github.com/m4rc1e/The-Lobster-Font

m4rc1e committed:
fonts | sources: changed version number from 2.000 to 2.001.

Increase needed because we fixed issue #525, https://github.com/google/fonts/issues/525

m4rc1e committed:
fonts | sources: fixed RFN. Non ascii characters were used in the copyright string

m4rc1e commented:
@alexeiva wouldn't it be better to pr this to @impallari's repo?

m4rc1e commented:
Alright, I've just done a quick review of this pr

---

![screen shot 2017-10-23 at 11 38 21](https://user-images.githubusercontent.com/7525512/31884935-bdd61830-b7e6-11e7-9d5e-1b68f4b4402c.png)
![screen shot 2017-10-23 at 11 37 30](https://user-images.githubusercontent.com/7525512/31884936-bdfbe89e-b7e6-11e7-818a-24f3d54dd27c.png)

Are these the correct shape?

They usually have a base which looks like this

![screen shot 2017-10-23 at 11 40 10](https://user-images.githubusercontent.com/7525512/31884997-f3847526-b7e6-11e7-9d4f-973e80b4fe7b.png)


---

These are missing 

![screen shot 2017-10-23 at 11 36 34](https://user-images.githubusercontent.com/7525512/31884960-cf951fc6-b7e6-11e7-8d7e-3aac5db200c7.png)

---

The quote right has gotten smaller

![lobster-quote-single](https://user-images.githubusercontent.com/7525512/31884968-d8544b14-b7e6-11e7-86fb-e3f7a9c5131e.gif)

alexeiva commented:
> @alexeiva wouldn't it be better to pr this to @impallari's repo?

I think this PR is a duplicate of https://github.com/impallari/The-Lobster-Font/pull/5. It's fine to close it.


Thanks for the review, I will get this fixed in the current expansion round.

`ғ` U+0493 can have a handwritten form too as in the case. One of the early adopters of this approach is Lucas' Calibri font.

`cyrbreve` and `cyrBreve` are obsolete (Adobe PUA U+F6D6 and U+F6D4). At the moment we use unencoded `brevecomb-cy` and `brevecomb-cy.case` for this.

To sum up, here is todo :

— add `commaaccent`
— fix `quoteright`

alexeiva committed:
added legacy commaaccent and fixed quoteright
as mentioned in https://github.com/ThomasJockin/The-Lobster-Font/pull/4

alexeiva commented:
Here is a historical sample of base glyph `г` and  `ӷ` `U+04F7` handrwitten

<img width="244" alt="screen shot 2017-10-23 at 4 42 41 pm" src="https://user-images.githubusercontent.com/16634089/31889557-4deecab4-b811-11e7-927d-d7d476bb9aa4.png">

alexeiva committed:
Latin-B update started

alexeiva committed:
added ⁒

alexeiva committed:
added ′″

alexeiva committed:
added gstroke.end

alexeiva committed:
added ezh.end

alexeiva committed:
added OT code for ezh.end and gstroke.end

alexeiva committed:
added contributors

alexeiva committed:
bumped version to 2.100

alexeiva committed:
fixed double component in minute

alexeiva committed:
corrected path in all layers

alexeiva committed:
removed non-exportable DEV cyr glyphs

alexeiva committed:
exported fonts v.2.100

alexeiva committed:
updated copyright

alexeiva committed:
removed stray glyph

alexeiva committed:
updated OT code

alexeiva committed:
removed stray glyph

alexeiva committed:
regenerated fonts

m4rc1e commented:
@ThomasJockin could you merge this please?
